packages/components/src/index.ts

```diff
@@ -210,6 +210,8 @@ export * from './Slide';
 
 export * from './Stepper';
 
+export * from './ScrollToTopButton';
```

Comment:
```suggestion
export { ScrollToTopButton, type ScrollToTopButtonProps }  from './ScrollToTopButton';
```

Пока еще нигде не зафиксировано, но для новых пакетов указываем явно экспортируемые сущности. Тем самым четко контролируем что экспортируется из пакета